lang/fr-FR.js

```diff
@@ -0,0 +1,3 @@
+export default {
+    welcome: 'Bienvenue'
+}
```

Comment:
Use JSON instead of JS for the translation files.

Reply:
Resolved